examples/crd_derive.rs

```diff
@@ -17,6 +17,7 @@ use serde::{Deserialize, Serialize};
     plural = "fooz",
     root = "FooCrd",
     namespaced,
+    description = "Custom resource representing a Foo",
```

Comment:
Oh, actually. Do you mind adding a line for docs about this in the lib.rs file in kube-derive? All the properties are documented there normally.

Reply:
Sure thing! Just wasn't aware of the docs. Added in 7754d4a9026cf02b22bcee65eaaea983714ee59b

Reply:
Sorry, forgot to sign :facepalm: It's now a32e7cc65bb539a767860600c0337dbdede3d7bd